ailab/benchmark/migrations/0001_initial.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-02-19 20:21
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Device',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('device', models.CharField(max_length=64, null=True)),
+                ('status', models.CharField(choices=[('AVA', 'AVAILABLE'), ('OCC', 'OCCUPIED'), ('OFF', 'OFFLINE'), ('DIS', 'DISABLED')], max_length=9, null=True)),
+                ('claimer', models.CharField(max_length=40, null=True)),
+                ('hash', models.CharField(max_length=32, null=True, unique=True)),
+                ('update_time', models.DateTimeField(null=True)),
+                ('heartbeat_time', models.DateTimeField(null=True)),
+                ('job_queue', models.CharField(default='pep_interactive', max_length=40)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='Run',
```

Comment:
database name?

Reply:
Removed the file and added migrations files to gitignore.